### PR TITLE
Add explicit dependency on new google auth library and bump existing …

### PR DESCRIPTION
**Pull request by deflaux**
…auth dependencies.

### CODE REVIEW

**Comment by dionloy** [comment]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -294,7 +304,8 @@
 
   <properties>
     <junit.version>4.11</junit.version>
-    <google.api.version>1.19.0</google.api.version>
+    <google.api.version>1.22.0</google.api.version>
+    <google.auth.version>0.4.0</google.auth.version>
     <google.api.genomics.version>v1-rev73-1.21.0</google.api.genomics.version>
```
Comment (pgrosu) [inline pom.xml]: v1-rev75-1.21.0 ?

File: pom.xml [hunk 2]
```diff
@@ -294,7 +304,8 @@
...
     <io.grpc.version>0.14.0</io.grpc.version>
```
Comment (pgrosu) [inline pom.xml]: 0.15.0-SNAPSHOT ?

**Comment by pgrosu** [comment]
I love the new streamlined approach!

**Pull request merged**